tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "paths": {
+      "@wagmi/chains": ["node_modules/@wagmi/chains/dist"],
```

Comment:
would it work if this path was for `@wagmi/chains` in `packages/core/node_modules`? that way we don't need to install `@wagmi/chains` again in the workspace root.